storage_controller/src/scheduler.rs

```diff
@@ -305,7 +305,7 @@ impl std::ops::Add for AffinityScore {
 
 /// Hint for whether this is a sincere attempt to schedule, or a speculative
 /// check for where we _would_ schedule (done during optimization)
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
For the code paths using this, they're already within the Clone impl of another struct (& I generally prefer to avoid Copy apart from really foundational stuff like IDs, to keep our memory copies explicit + visible)